webpack/components/Search/Search.js

```diff
@@ -39,9 +39,14 @@ class Search extends Component {
     ];
 
     if (autoCompleteParams[0] !== '') {
-      const { data } = await api.get(...autoCompleteParams);
+      let data;
+      if (this.props.foremanApiAutoComplete) {
+        data = await foremanApi.get(...autoCompleteParams);
+      } else {
+        data = await api.get(...autoCompleteParams);
+      }
```

Comment:
what happens if there is an api error here? 
Does it send an error toast? 
Would data then be undefined? 
If so, should we do a falsey check below (data?.data?.filter) to prevent the error screen of death?
or would we want to set some alternative state?

Reply:
When me and Jeremy were working on this together this is what he suggested, I don't think we were thinking about errors yet. I can add another check here

Reply:
you could wrap this in an try catch statement, or if the api is handling the toast error, you can simply just add a check to the setState to make sure you are just setting undefined instead of hitting another error there that would block the app.

Reply:
Added the `data?.data?.filter` and updated two other places I found `Tablewrapper` needing `actionButtons` it looks like other tests are failing still but I searched and cannot find any more instances of places that need `actionButtons` set